.env.docker.dev

```diff
@@ -1,7 +0,0 @@
-# Environment variables for use with development in Docker
-FREQUENCY_URL=ws://host.docker.internal:9944
-PROVIDER_ID=1
-PROVIDER_BASE_URL=https://some-provider
-REDIS_URL=redis://redis:6379
-PROVIDER_USER_GRAPH_ENDPOINT=user-graph
-PROVIDER_ACCESS_TOKEN=some-token
```

Comment:
Should we leave this to help with docker needed hosts in dev for `REDIS` and `FREQUENCY_URL`?